core/src/main/java/io/aiven/klaw/model/enums/PermissionType.java

```diff
@@ -22,6 +22,7 @@ public enum PermissionType {
   SYNC_BACK_TOPICS("To Synchronize topics back to cluster"),
   SYNC_BACK_SUBSCRIPTIONS("To Synchronize subscriptions back to cluster"),
   SYNC_CONNECTORS("To Synchronize Kafka connectors From Cluster"),
+  SYNC_SCHEMAS("To Synchronize schemas From Cluster"),
 
```

Comment:
Do we need to add a bit in the release notes to add this permission to the super user(if they want to) or are we adding it automatically?

Reply:
It's ok to add directly to SUPERADMIN, similar to SYNC_TOPICS. Updated default permissions

Reply:
We shall update rel notes too